### PR TITLE
Xenomorph Hybrids species

### PR DESCRIPTION
**Pull request by Aroliacue**
:cl:
add: Added in a new Xenomorph Hybrid species, with unique stats and Xenomorph-centric abilities.
add: Added Xenolingua and Hivemind as selectable languages. Hivemind requires you to be a Xenomorph Hybrid.
qol: Gave Zaddat the ability to move much faster with their special Shroud voidsuits.
/:cl:

### CODE REVIEW

File: code/modules/mob/language/outsider.dm [hunk 1]
```diff
@@ -77,17 +77,17 @@
 /datum/language/xenocommon
 	name = "Xenolingua" //CHOMPedit
 	colour = "alien"
-	desc = "The common tongue of both the xenomorphs and the Genaprawns." //CHOMPedit
+	desc = "The common tongue of both the Xenomorphs and the Genaprawns." //CHOMPedit
```
Comment (Darlantanis) [inline code/modules/mob/language/outsider.dm:80]: If "xenomorph" is going to be a species name, it's not a proper noun. Lowercase.

File: code/modules/mob/language/outsider.dm [hunk 2]
```diff
@@ -77,17 +77,17 @@
...
 	exclaim_verb = "hisses"
 	key = "u"
-	flags = RESTRICTED
+// 	flags = RESTRICTED // CHOMPedit: Anyone can use it.
 	syllables = list("sss","sSs","SSS")
 
 /datum/language/xenos
 	name = "Hivemind"
-	desc = "Some aliens have the strange ability to commune over a psychic hivemind." //CHOMPedit
+	desc = "A psychic link permitting members of a Xenomorph hive to communicate over enormous distances." //CHOMPedit
```
Comment (Darlantanis) [inline code/modules/mob/language/outsider.dm:90]: If "xenomorph" is going to be a species name, it's not a proper noun. Lowercase.

File: modular_chomp/code/modules/mob/living/carbon/human/species/station/station_ch.dm [hunk 3]
```diff
@@ -47,4 +47,189 @@
...
+	slowdown = -0.2 // Slightly faster than average.
+	total_health = 150 // Takes a lot of effort to take a Xenomorph down.
+	brute_mod = 0.90 // Physical damage doesn't phase them as much.
+	burn_mod = 1.40 // They do NOT like fire.
+	toxins_mod = 0.90 // Slightly resistant to toxins.
+	radiation_mod = 0.7 // Quite resistant to radiation exposure.
+	flash_mod = 1.1 // Flashes are slightly more effective.
+	metabolic_rate = 1.2 // Very physically active species, thus requiring more nutritional intake.
+	item_slowdown_mod = 0.90 // They carry heavy things slightly better.
+	mob_size = MOB_LARGE
```
Comment (Darlantanis) [inline modular_chomp/code/modules/mob/living/carbon/human/species/station/station_ch.dm:73]: Keep the default mob_size. Changing this on carbons is fucky and likely to have many unintended consequences.

Comment (Aroliacue) [reply modular_chomp/code/modules/mob/living/carbon/human/species/station/station_ch.dm]: Changed to MOB_MEDIUM

File: code/modules/mob/language/outsider.dm (hunk 2 above)
Comment (Aroliacue) [reply code/modules/mob/language/outsider.dm]: Corrected

**Comment by Aroliacue** [comment]
Ready for Review / Merge

File: code/modules/materials/materials/organic/resin.dm [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /datum/material/resin
 	name = "resin"
-	icon_colour = "#35343a"
+	icon_colour = "#353C40"
```
Comment (Raeschen) [inline code/modules/materials/materials/organic/resin.dm:3]:
```suggestion
	icon_colour = "#353C40" //ChompEDIT
```

**Review by Raeschen — CHANGES_REQUESTED** [review]
ONE chompedit, ehehe

**Pull request merged**